### ISSUE TITLE
Cannot export TorchScript model after git pull

### ISSUE THREAD

**Issue by srn-source** [ultralytics/yolov5#9215]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Export

### Bug

I am using git pull update my project, I got this when I try to export TorchScript model but for other model like Tflite can export normally.

what should I do?

export: data=D:\my_data\yolov5\data\custom_data.yaml, weights=['runs\\train\\exp29\\weights\\best.pt'], imgsz=[1280], batch_size=1, device=cpu, half=False, inplace=False, train=False, keras=False, optimize=True, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.4, conf_thres=0.4, include=['torchscript']
YOLOv5  v6.2-78-g3a17ba4 Python-3.9.7 torch-1.9.0+cu111 CPU

Fusing layers... 
Model summary: 280 layers, 3089188 parameters, 0 gradients, 4.2 GFLOPs

PyTorch: starting from runs\train\exp29\weights\best.pt with output shape (1, 102000, 7) (6.6 MB)

TorchScript: starting export with torch 1.9.0+cu111...
TorchScript: export failure  2.2s: Inplace update to inference tensor outside InferenceMode is not allowed.You can make a clone to get a normal tensor before doing inplace update.See https://github.com/pytorch/rfcs/pull/17 for more details.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@srn-source 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. Torchscript export works correctly in my tests:

<img width="1246" alt="Screenshot 2022-08-30 at 15 43 28" src="https://user-images.githubusercontent.com/26833433/187453148-88d7b401-5cba-4f48-aa94-613475e11703.png">

We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by worklikerlei** [comment]
Hello, I also encountered this problem. How did you solve it？

**Comment by tongchangD** [comment]
> Hello, I also encountered this problem. How did you solve it？

me too

**Comment by glenn-jocher** [comment]
@tongchangD and @srn-source, thanks for reaching out! We appreciate your patience on this matter. The Ultralytics team is currently investigating the TorchScript export failure issue with YOLOv5 and we will keep you updated on any progress made. In the meantime, we encourage you to follow the guidelines mentioned in the previous response to provide us with a minimum reproducible example if you're experiencing issues with the TorchScript export functionality. This will help us better understand and address the problem. Thank you for your understanding and cooperation!
